src/Host/Client/Impl/Host/LocalBrokerClient.cs

```diff
@@ -24,14 +24,15 @@ internal sealed class LocalBrokerClient : BrokerClient {
         private const string InterpreterId = "local";
 
         private static readonly bool ShowConsole;
-        private static readonly LocalCredentialsDecorator Credentials = new LocalCredentialsDecorator();
+        private static readonly LocalCredentialsDecorator _credentials = new LocalCredentialsDecorator();
```

Comment:
VS showed me that it must be `Credentials`. What is our convention for `private static readonly`?